src/languages/javascript.js

```diff
@@ -256,15 +256,15 @@ export default function(hljs) {
   const PARAMS_CONTAINS = SUBST_AND_COMMENTS.concat([
     // eat recursive parens in sub expressions
     {
-      begin: /\(/,
+      begin: /(\s*)\(/,
       end: /\)/,
       keywords: KEYWORDS,
       contains: ["self"].concat(SUBST_AND_COMMENTS)
     }
   ]);
   const PARAMS = {
     className: 'params',
-    begin: /\(/,
+    begin: /(\s*)\(/, // to match the parms with 
```

Comment:
```suggestion
    // convert this to negative lookbehind in v12
    begin: /(\s*)\(/, // to match the parms with 
```

Reply:
do you mean to remove this change or add this `// convert this to negative lookbehind in v12` comment above the begin?
I am sorry, i can't understand you.

Reply:
Just add the comment.  I wouldn't have suggested the change otherwise.

Reply:
Ok, that's nice to add this comment.
Thank you